COLLECTION.md

```diff
@@ -6,6 +6,7 @@
 
 |  #  | Brand              | Type        | Name                        | Origin      | Flavour                     |
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------------------|
+| 142 | [Jia ming]         | [Green]     | [West Lake Dragon Well]     | [China]     | Green tea                   |
```

Comment:
[West Lake Dragon Well] - this link is used by another tea. Please pick another name and create a link.